File: jexler-war/src/main/webapp/WEB-INF/jexlers/filetailer.rb
import Java::net.jexler.StopService
import Java::net.jexler.FileTailerService

s = FileTailerService.new($jexler, "selftailer")
s.setFile($file.getAbsolutePath)
s.addFilterPattern("^import")
s.start
$services.add s

begin
  event = $events.take
  if event.is_a? FileTailerService::Event
    puts "Got line (ruby): " + event.getLine
  elsif event.is_a? StopService::Event
    return
  end
end while true

File: jexler-war/src/main/webapp/WEB-INF/jexlers/filetailer.py
# autostart
from net.jexler import StopService
from net.jexler import FileTailerService

s = FileTailerService(jexler, "selftailer")
s.setFile(file.getAbsolutePath())
s.addFilterPattern("^from")
s.start()
services.add(s)

while True:
    event = events.take()
    if isinstance(event, FileTailerService.Event):
        print "Got line (jython): %s" % event.getLine()
    elif isinstance(event, StopService.Event):
        break
